mne/viz/_3d.py

```diff
@@ -2602,22 +2531,22 @@ def plot_sparse_source_estimates(src, stcs, colors=None, linewidth=2,
     from .backends.renderer import _get_renderer
 
     linestyles = [
-        ('solid',                 'solid'),
-        ('dashed',                'dashed'),
-        ('dotted',                'dotted'),
-        ('dashdot',               'dashdot'),
-        ('loosely dotted',        (0, (1, 10))),
-        ('dotted',                (0, (1, 1))),
-        ('densely dotted',        (0, (1, 1))),
-        ('loosely dashed',        (0, (5, 10))),
-        ('dashed',                (0, (5, 5))),
-        ('densely dashed',        (0, (5, 1))),
-        ('loosely dashdotted',    (0, (3, 10, 1, 10))),
-        ('dashdotted',            (0, (3, 5, 1, 5))),
-        ('densely dashdotted',    (0, (3, 1, 1, 1))),
-        ('dashdotdotted',         (0, (3, 5, 1, 5, 1, 5))),
-        ('loosely dashdotdotted', (0, (3, 10, 1, 10, 1, 10))),
-        ('densely dashdotdotted', (0, (3, 1, 1, 1, 1, 1)))
+        ('solid',                 'solid'),                     # noqa: E241
+        ('dashed',                'dashed'),                    # noqa: E241
+        ('dotted',                'dotted'),                    # noqa: E241
+        ('dashdot',               'dashdot'),                   # noqa: E241
+        ('loosely dotted',        (0, (1, 10))),                # noqa: E241
+        ('dotted',                (0, (1, 1))),                 # noqa: E241
+        ('densely dotted',        (0, (1, 1))),                 # noqa: E241
+        ('loosely dashed',        (0, (5, 10))),                # noqa: E241
+        ('dashed',                (0, (5, 5))),                 # noqa: E241
+        ('densely dashed',        (0, (5, 1))),                 # noqa: E241
+        ('loosely dashdotted',    (0, (3, 10, 1, 10))),         # noqa: E241
+        ('dashdotted',            (0, (3, 5, 1, 5))),           # noqa: E241
+        ('densely dashdotted',    (0, (3, 1, 1, 1))),           # noqa: E241
+        ('dashdotdotted',         (0, (3, 5, 1, 5, 1, 5))),     # noqa: E241
+        ('loosely dashdotdotted', (0, (3, 10, 1, 10, 1, 10))),  # noqa: E241
+        ('densely dashdotdotted', (0, (3, 1, 1, 1, 1, 1)))      # noqa: E241
     ]
```

Comment:
I find the diff in this file super hard to read. Can you explain what you did? thx

Reply:
I think I modified the file in some way that I got a flake error when it was recomputed without ignoring this part so I just did this to satisfy the CIs

Reply:
my comment here was concerning the diff of the entire file. What was the big picture logic? add such a function like _plot_sensors that takes a renderer param so Brain and plot_alignment can use the same code to display sensors?